docs/DefiningMockdata.md

```diff
@@ -0,0 +1,18 @@
+# Defining mockdata
+
+
+## Influencing entity set behavior
+
+The mockserver allows you to define your mock data as javascript file and function that allow you to influence the behavior of the standard function to match your needs.
+
+In order to define your mockdata, you need to create a file with the name of the entity set you want to mock and the extension `.js` in the folder identifier by your `mockdataPath`.
+
+You can then implement any of the function defined in the [API](./MockserverAPI.md) to add your custom logic.
```

Comment:
> any of the function

"some of the functions"? I think "any" means "all of them"

Reply:
`some functions` seems to be a good alternative